### ISSUE TITLE
https://twentyone.world/group is 404

### ISSUE THREAD

**Issue by Fiach-Dubh** [twentyone-world/twentyone-world.github.io#36]
https://twentyone.world/group is 404

**Comment by NatoshiSakamot** [comment]
Yep.  Unsure how to add new group.  https://bitcoinevents.uk/ is the co-ordination place for all UK & Ireland meet up groups, of which there are now 40+.  We would like to link this up with twentyone.world if poss and this link seemed to be the place to do it, but 404 error.  Will raise an issue.
